packages/application-shell/src/utils/setup-global-error-listener/setup-global-error-listener.ts

```diff
@@ -22,15 +25,14 @@ export default function setupGlobalErrorListener() {
           'handled. This is most likely a bug in the software. Please ensure ' +
           'that the promise is correctly handled.'
       );
-    internalReduxStore.dispatch(
-      showUnexpectedErrorNotification({ error: { message: event.reason } })
```

Comment:
This was actually wrong! We didn't do anything with the `error`, as the notification only accepts a `errorId`.
Now we report the error to sentry and pass the sentry `errorId`.